gordo_components/builder/build_model.py

```diff
@@ -45,10 +46,13 @@ def build_model(
         Mapping of the Dataset to initialize, following the same logic as model_config.
     metadata: dict
         Mapping of arbitrary metadata data.
+    cv_mode: 
```

Comment:
`cv_mode: str` 
But also wondering if maybe this is a place where we should be using an `Enum`? 

```python
CvMode = Enum('CvMode', 'FULL PARTIAL SKIP')

# Probably done in the CLI portion:
cv_mode = getattr(CvMode, cv_mode_from_cl_or_click_type_param, CvMode.FULL)

# Then this parameter's signature is:
cv_mode: CvMode
```

Because even in the documentation, without looking at the code, I don't know what the possible values of `cv_mode` are; having it as an `Enum` keeps it pretty explicit about the options, and can even auto update it in `click` as an option with `CvMode.__members__`
```

Reply:
@epa095 probably has input on this. :moneybag: